lib/manager/components/LabelEditor.js

```diff
@@ -209,7 +204,7 @@ class LabelEditor extends React.Component<Props, State> {
             type='submit'
             onClick={this._onSaveSettings}
           >
-              Save
+            Save
```

Comment:
Is it supposed to trigger a full page refresh? @landonreed what do you think?

Reply:
It used to only refresh feed sources, but then at Landon's suggestion we had it refresh project deployments as well which causes the full page refresh. I am not sure why we fetch project deployments, but I trust Landon since he's worked with this a lot longer than I have